hadoop-dist/pom.xml

```diff
@@ -29,6 +29,13 @@
   <name>Apache Hadoop Distribution</name>
   <packaging>jar</packaging>
 
+  <properties>
+    <file.encoding>UTF-8</file.encoding>
+    <downloadSources>true</downloadSources>
+    <docker.hadoop-runner.version>docker-hadoop-3</docker.hadoop-runner.version>
```

Comment:
No idea why `apache/hadoop-runner` has a version tagged `docker-hadoop-3`, but it is the oldest and largest one.  I don't think it should be used.  `jdk8` would be a safe choice.

https://hub.docker.com/r/apache/hadoop-runner/tags